modules/generic-scala3/src/main/scala-3/pureconfig/generic/derivation/HintsAwareCoproductConfigReaderDerivation.scala

```diff
@@ -0,0 +1,52 @@
+package pureconfig
+package generic
+package derivation
+
+import scala.compiletime.*
```

Comment:
Same thing as above - please maintain the same import style as in the rest of the repository.

```suggestion
import scala.compiletime._
```

Reply:
Sure, I can change it!

But let's keep in mind that [Scala 3 reference](https://docs.scala-lang.org/scala3/reference/changed-features/imports.htm) mentions the `_` imports as something to  `dropped in a future versions`, so I think we should treat it more as a `givens` / `using` rather than optional braces and at some point commit to `*`.

Reply:
Yeah, I saw that the latest Scala 3 version already treats it as deprecated... I'm curious to see how it affects projects targeting Scala 2 too, though. We'll continue cross-compiling `core` with Scala 2 and 3 for the foreseeable future, which prevents us from adopting Scala 3 specific syntax there.

Reply:
There are `-Xsource:3` and `-Xsource:3-cross` [flags](https://docs.scala-lang.org/scala3/guides/migration/tooling-scala2-xsource3.html) which can help with that, but only for Scala 2.13, so I guess it can be leveraged only if pureconfig drops Scala 2.12.